lib/fun_with_flags/protocols/group.ex

```diff
@@ -65,7 +65,7 @@ defprotocol FunWithFlags.Group do
       iex> FunWithFlags.Group.in?(data, :superusers)
       false
   """
-  @spec in?(term, atom) :: boolean
+  @spec in?(term, atom | String.t) :: boolean
```

Comment:
Can we make this `String.t | atom` please? Mainly because this is really meant to be a binary, and atoms are supported for retro-compatibility.

Also, the documentation comment above this function still uses atoms (`:staff`, `:superusers`). It would be great if you could make them binaries too.

Reply:
Sorry for the delay. Got it updated. What do you think?

Reply:
Perfect, thanks!